exercise-solutions/quiz-game/part-03/src/index.jsx

```diff
@@ -24,21 +24,28 @@ class App extends React.Component {
 
 
     renderAnswerTag(prefix, answer, correct) {
-        return <div className='gameBtn' onClick={() => this.handleClick(correct)}> {prefix + answer} </div>;
+        return <div className='answer' onClick={() => this.handleClick(correct)}> {prefix + answer} </div>;
     }
 
     render() {
 
         const quiz = this.state.quiz;
 
         return (
-            <div>
+            /*This <></> is a shorthand React fragment, full tag is <React.Fragment></React.Fragment>, 
```

Comment:
@arcuri82 can you please look at the spelling here? <3

Reply:
I ll change this text directly in a new commit

Reply:
Ok, you merge once you are done with that? =)